realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -714,6 +713,7 @@ public void clear() {
         }
     }
 
+    // FIXME: This stuff doesn't appear to be used.  It should be deleted or explained.
```

Comment:
Not removing that comment until it can be replaced with a description of the use of that stuff!  ;-P